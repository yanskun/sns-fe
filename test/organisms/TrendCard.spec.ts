import { shallowMount } from '@vue/test-utils'
import Component from '~/components/organisms/TrendCard.vue'

describe('TrendCard', () => {
  test('snapshot', () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        tags: [
          { title: 'tag1', color: 'red', textColor: 'white' },
          { title: 'tag2', color: 'blue', textColor: 'white' },
        ],
        comment: 'test',
        org: {
          href: 'https://github.com/',
          src: 'https://github.githubassets.com/images/modules/open_graph/github-logo.png',
          title: 'GitHub',
          description:
            'GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects.',
        },
      },
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
