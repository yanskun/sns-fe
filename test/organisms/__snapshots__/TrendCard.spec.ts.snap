// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrendCard snapshot 1`] = `
<v-card-stub
  loaderheight="4"
  tag="div"
>
  <template>
    <v-progress-linear-stub
      active="true"
      buffervalue="100"
      color="primary"
      height="10"
      indeterminate="true"
      value="0"
    />
  </template>
  <v-card-text-stub>
    <v-chip-stub
      active="true"
      activeclass=""
      class="tag"
      closeicon="$delete"
      closelabel="$vuetify.close"
      color="red"
      filtericon="$complete"
      tag="span"
      textcolor="white"
    >
      tag1
    </v-chip-stub>
    <v-chip-stub
      active="true"
      activeclass=""
      class="tag"
      closeicon="$delete"
      closelabel="$vuetify.close"
      color="blue"
      filtericon="$complete"
      tag="span"
      textcolor="white"
    >
      tag2
    </v-chip-stub>
    <p
      class="comment"
    >
      test
    </p>
    <v-card-text-stub>
      <org-card-stub
        description="GitHub is where people build software. More than 65 million people use GitHub to discover, fork, and contribute to over 200 million projects."
        href="https://github.com/"
        src="https://github.githubassets.com/images/modules/open_graph/github-logo.png"
        title="GitHub"
      />
    </v-card-text-stub>
  </v-card-text-stub>
</v-card-stub>
`;
